.github/workflows/add-to-triage.yml

```diff
@@ -0,0 +1,18 @@
+name: Add to Triage
+
+on:
+  issues:
+    types:
+      - opened
+
+jobs:
+  add-to-project:
+    name: Add issue to project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@v0.4.0
+        with:
+          project-url: https://github.com/orgs/eslint/projects/3
+          github-token: ${{ secrets.PROJECT_BOT_TOKEN }}
```

Comment:
`secrets.PROJECT_BOT_TOKEN` does not appear to be available to this repository, is it intentional?

Reply:
I've added `espree`, `eslint-scope` and `generator-eslint` to the list of repositories that can use `secrets.PROJECT_BOT_TOKEN`.